README.md

```diff
@@ -38,12 +42,18 @@ Python](https://www.python.org/dev/peps/pep-0020/).
 
 ## 1. Formality-Core
 
-Formality-Core is the minimal core behind Formality. If you see Formality as a
-programming language, then Core is the minimal amount of features required to
-derive everything else as libraries. If you see it as a proof language, then
-Core is the set of axioms from which all of mathematics derive. On this section,
-we'll specify it informally. Reference implementations using popular languages
-are provided later. 
+Formality-Core is the minimal semantic core behind Formality. If you look at
+Formality as a programming language, then Core is the minimal amount of features
+required to derive everything else as libraries. If you look at it as proof
+language, then Core is the set of axioms underlying the proof system. The
+[Curry-Howard
+correspondence](https://en.wikipedia.org/wiki/Curry%E2%80%93Howard_correspondence)
+describes both perspectives as essentially identical and implies that, in
+principle, Formality is capable of expressing any mathematical statement.
```

Comment:
Too abstract, a regular person/developer will read that sentence and get no information from it. A few more of those and he/she will feel like the text wasn't meant for him/her, get bored and disconnected and close it. Let's avoid sentences like that. Let talk simple, without jargons, in a way that a child could understand. Most of the concepts here should absolutely be explainable on their own. "If you can't explain it simple, you don't understand it well enough." Let's make an effort to replace obscure jargons by simple words.

Reply:
Okay, I will try to think of a more straightforward way to describe proofs as programs